src/redprl/operator.sml

```diff
@@ -483,6 +483,8 @@ struct
              ^ "["
              ^ dirToString f dir
              ^ "]"
+       | CUST (opid, [], _) => 
+           f opid
        | CUST (opid, ps, _) =>
            f opid ^ "{" ^ paramsToString f ps ^ "}"
```

Comment:
@favonia That's defined locally in some other module, I think. I guess we could define it here, but it is probably not worth it.